### DISCUSSION TITLE
[help] alt + arrows works fine but hjkl doesnt move to next pane

### DISCUSSION THREAD

**Discussion by dvrd** [zellij-org/zellij#2398] [Q&A]
I've been trying to figure out what is the issue but I can't seem to find a way to make zellij recognize `Alt + hjkl` to move. Any recommendations to sort this one out?

**Comment by azinsharaf** [comment]
have you mapped alt + hjkl in another program? For instance i've mapped it in GlazeWM window manager.

**Comment by Lockszmith-GH** [comment]
What environment are you running in?
Linux / MacOS / Windows ?
What terminal emulator applicationare you running?
